packages/liveblocks-react-ui/src/components/Comment.tsx

```diff
@@ -110,6 +111,11 @@ export interface CommentProps extends ComponentPropsWithoutRef<"div"> {
    */
   showAttachments?: boolean;
 
+  /**
+   * Whether to show the composer's formattng controls when editing the comment.
+   */
+  showComposerFormattingControls?: ComposerProps["showFormattingControls"];
```

Comment:
I'm naming this "formatting controls" instead of being specific to the toolbar because we'll re-use it for other floating controls like the link edition UI, etc.